lib/IRGen/GenReflection.cpp

```diff
@@ -1007,6 +1007,104 @@ void IRGenModule::emitBuiltinTypeMetadataRecord(CanType builtinType) {
   builder.emit();
 }
 
+ // TODO: This is copied from GenEnum.cpp; should we refactor to eliminate
```

Comment:
TODO

Reply:
@aschwaighofer - Copying this function from GenEnum.cpp was helpful.  Should I refactor to share one copy of this function?  Or do you prefer to keep utilities like this local to the source file they're used in?

Reply:
@tbkka I would make it available via `GenEnum.h`

Reply:
Seems to be in GenValueWitness.cpp now; I exposed it via GenValueWitness.h so there's no longer a separate copy here.